### PR TITLE
PR for e-commerce home screen!

### PR DESCRIPTION
**Pull request by adidoshi**
### Hey I created home screen for e-commerce store, please review.
* I've also tried to make the page responsive.

### CODE REVIEW

**Review by preethasri — APPROVED** [review]
Looks neat and clean

File: style.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
+/* header css */
+.navigation {
+  border: none;
+  padding: 1rem 5rem;
+  position: sticky;
+  z-index: 9;
+  left: 0;
+  top: 0;
+  background-color: #fcfcfc;
```
Comment (msdeshmukh009) [inline style.css:9]: color variable maybe?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+#slider figure {
+  position: relative;
+  width: 500%;
+  margin: 0;
+  left: 0;
+  animation: 20s slider infinite;
+}
+
+#slider figure img {
```
Comment (msdeshmukh009) [inline style.css:47]: Maybe avoid id selectors for styling.

**Review by msdeshmukh009 — APPROVED** [review]
The UI is looking really good!!

File: style.css (hunk 1 above)
Comment (adidoshi) [reply style.css:9]: Yes I'll make a variable0. Thankyou

File: style.css (hunk 2 above)
Comment (adidoshi) [reply style.css:47]: Yes, I agree, but actually for the slider animation purpose, I used it.

**Pull request merged**